src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -1,12 +1,13 @@
 package core.mate.academy.service;
 
 import core.mate.academy.model.Bulldozer;
+import core.mate.academy.model.Machine;
 import java.util.List;
 
-public class BulldozerProducer implements MachineProducer<Bulldozer> {
+public class BulldozerProducer implements MachineProducer<Machine> {
```

Comment:
why did we replace Bulldozer on Machine here?

Reply:
sorry, didn't notice. Yesterday we're reviewing this task with each other and someone has code like 
`public class BulldozerProducer implements MachineProducer<Machine>`. 
So I changed it into my code to get the screen-shot and ask Bohdan which option is correct.

fixed!